### ISSUE TITLE
[Enhancement]: check for PEP 584

### ISSUE THREAD

**Issue by pums974** [dosisod/refurb#254]
### Overview

It would be nice to detect pattern like `{**dict1, **dict2}`
and replace it with `dict1 | dict2`

I'm sorry, I wish I got some time to implement it and propose a PR, but I don't. Maybe later though.

### Proposal

The pattern could be one of the following form (+ repetitions)
 ```python
{**dict1, **dict2} # ->  dict1 | dict2
{k:v, **dict2} # ->  {k:v} | dict2
{**dict1, k:v} # ->  dict1 | {k:v}
```

**Comment by dosisod** [comment]
I think this would be a good thing to add! I'll see if I can implement this sometime next week, I've been pretty busy recently as well.

Also, don't feel bad about not having the time to write a PR for this! Writing up an issue is equally important as it lets me know what features people want.

**Comment by dosisod** [comment]
Closing, implemented this in b364ecfa74007fa6e48142b89572f7c7496402fd.

Thank you for submitting this issue! I'll publish a new release sometime tomorrow.
